x-pack/plugins/uptime/server/plugin.ts

```diff
@@ -101,6 +105,8 @@ export class Plugin implements PluginType {
       () => this.savedObjectsClient
     );
 
+    this.telemetryEventsSender.setup(plugins.telemetry);
```

Comment:
i think this should move under `isServiceEnabled` condition, i see no value of enabling if service isn't enabled.